client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,9 +97,21 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
-
-			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
+			const scheduledTime =
+				scheduledDate.diff( now, 'days' ) <= 7
+					? scheduledDate.calendar()
```

Comment:
Wouldn't `calendar()` always work, see to [documentation](https://momentjs.com/docs/#/customization/calendar/)? We might want to customize the `sameElse` entry to `LLL`.

Reply:
The issue I'm finding with that is it looks a bit odd since there's no "at" for the bottom post; I'm finding that although it can be added, I'm not sure it can be added as a translatable phrase. Is there a way around this? 

<img width="618" alt="Screenshot 2020-04-07 at 14 27 47" src="https://user-images.githubusercontent.com/43215253/78674979-4d1b6280-78dc-11ea-8a93-5a89ff45b968.png">